integration-tests/CodegenTests.test.scala

```diff
@@ -1,8 +1,8 @@
 package besom.integration.codegen
 
-import os./
 import besom.integration.common.*
 import munit.{Slow, TestOptions}
+import os.{/, *}
```

Comment:
`os.*` imports `os./` anyway. Not sure if we need `*` though as typically the `os.`prefix is always used

Reply:
```suggestion
import os.*
```